homework-2/scss/_variables.scss

```diff
@@ -0,0 +1,26 @@
+@import "font-family";
+
+$color-text-primery: #FFF;
+$bgc-primery: #FFF;
+$color-text-seconrary: #191919;
+$bgc-border-primery: #22BF95;
+$colot-text-third: #22BF95;
+$color-text-fourth: #7F7F7F;
+$color-text-fourth-light: #B2B2B2;
+$bgc-border-fourth-light: #B2B2B2;
+
+$full-width: 100%;
+
+$mobile: 374px;
+$tablet: 767px;
+$desktop: 1023px;
+$widescreen: 1439px;
```

Comment:
all these breakpoint values are correct for the desktop-first approach, for mobile-first it should be every value +1px